free-programming-books.md

```diff
@@ -274,7 +274,8 @@
 ### Graphics Programming
 
 * [3D Game Shaders For Beginners](https://github.com/lettier/3d-game-shaders-for-beginners) - David Lettier (Git) [(HTML)](https://lettier.github.io/3d-game-shaders-for-beginners)
-* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) - Wikibooks
+* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) (Wikibooks)
```

Comment:
Could you revert this, I believe Wikibooks was meant to be separated with a hyphen.
Please see other resources from Wikibooks on this page for examples.

```suggestion
* [Blender 3D: Noob to Pro](https://en.wikibooks.org/wiki/Blender_3D%3A_Noob_to_Pro) - Wikibooks
```